docs/nlu/choosing-a-pipeline.rst

```diff
@@ -99,8 +113,10 @@ create your own :ref:`custom tokenizer <custom-nlu-components>`.
     on the individual components in :ref:`components`. If a required component is missing inside the pipeline, an
     error will be thrown.
 
+
 Featurization
 ~~~~~~~~~~~~~
+
 You need to decide whether to use components that provide pre-trained word embeddings or not.
```

Comment:
@tabergma

Reply:
As ConveRTFeaturizer produces contextualized word embeddings, it takes the context (e.g. all words in the sentence) into account when creating the word embedding. So you should get different embeddings for a word depending on the context. 
However, ConveRT is trained on a specific dataset (Reddit), so it only knows the domains that are covered in that dataset. If that dataset does not contain any examples from the banking domain, it would not know that "balance" and "cash" are related. So, I would say the example is still valid.